gap/snksetswrsr.gi

```diff
@@ -29,7 +29,7 @@ SetInfoLevel( InfoJellyfish, 1 );
 ##
 ##  Computes all [n,k,r] such that N = Binomial(n,k)^r. Returns those
 ##  satisfying k*r > 1 and 2*r*k^2 < n (sufficient condition from
-##  BLS (Lemma 2.10) that ensures that there is a unique minimal and a
+##  BLS97 (Lemma 2.10) that ensures that there is a unique minimal and a
```

Comment:
Can you think of a place to (perhaps additionally) cite this paper in the auto doc, so that it appears in the manual?

Reply:
This looks very much like an implementation detail. I think there is no sensible place in the manual to reference it. :(